src/test_lib_json/main.cpp

```diff
@@ -307,7 +307,21 @@ JSONTEST_FIXTURE(ValueTest, arrayIssue252) {
   }
   // JSONTEST_ASSERT_EQUAL(5, root["array"].size());
 }
+JSONTEST_FIXTURE(ValueTest, arrayIssue691) {
+  Json::Value array2;
```

Comment:
why "array2"?

Reply:
Well, the `array1_` has been used above,  so I used `array2_`, but consdering the rule of no trailing underscore for local variables  I changed it to `array2`.

Reply:
My point was that there's no other local array, so the array2 makes me wonder where array1 went. Just simpler array or arr would be better, I think.

Reply:
Yep, it has been chaned to `array` in the former update.